### PR TITLE
Add endpoint of wallet /wallet/transaction/history/:address

### PR DESCRIPTION
**Pull request by MichaelKim20**
I added a new `/wallet/transaction/history/:address` to replace `/wallet/transactions/history/:address`.

### CODE REVIEW

**Review by Kang-Simon — APPROVED** [review]
LGTM

File: src/Stoa.ts [hunk 1]
```diff
@@ -891,6 +896,162 @@ class Stoa extends WebService {
...
+     *      default is "inbound,outbound,freeze,payload"
+     * The parameter `beginDate` is the start date of the range of dates to look up.
+     * The parameter `endDate` is the end date of the range of dates to look up.
+     * The parameter `peer` is used when users want to look up only specific
+     *      address of their counterparts.
+     *      Peer is the withdrawal address in the inbound transaction and
+     *      a deposit address in the outbound transaction
+     * Returns a set of transactions history of the addresses.
+     * ```
+     */
```
Comment (TrustHenry) [inline src/Stoa.ts:927]: Nice!

**Review by TrustHenry — APPROVED** [review]
LGTM

File: src/Stoa.ts [hunk 2]
```diff
@@ -252,6 +252,11 @@ class Stoa extends WebService {
             isBlackList,
             this.getWalletTransactionsHistory.bind(this)
         );
+        this.app.get(
+            "/wallet/transaction/history/:address",
+            isBlackList,
+            this.getWalletTransactionHistory.bind(this)
+        );
```
Comment (TrustHenry) [inline src/Stoa.ts:259]: /wallet/transactions/history/:address <- Are you going to remove this endpoint later?
If so, it's good to write a comment to remove it later.

Comment (MichaelKim20) [reply src/Stoa.ts:259]: I added `@deprecated`

**Pull request merged**